### PR TITLE
Meteor 1.6.0.1

### PR DESCRIPTION
**Pull request by aaronjudd**
METEOR 1.6.0.1

### CODE REVIEW

**Comment by jshimko** [comment]
Thanks!  

I have some plans to refactor the main Dockerfile here so that the alternate dev Dockerfile will no longer be relevant.  That's because we can now do [multi-stage builds with a single Dockerfile](https://docs.docker.com/engine/userguide/eng-image/multistage-build/) and that allows the efficient caching of our dev build without all of the image size cost.  Coming soon...

In the meantime, looks like CircleCI isn't loving the Meteor 1.6.1 download yet.

**Comment by jshimko** [comment]
Also, it's Meteor 1.6.0.1, not 1.6.1.  Pretty sure that should explain CircleCI's failure.

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## v3.0.0
 
-- Meteor 1.6
+- Meteor 1.6.1
```
Comment (jshimko) [inline HISTORY.md]: We need to bump the version of the image.  We can't just add this to a previous release.

Comment (jshimko) [reply HISTORY.md]: And this should be 1.6.0.1

**Review by jshimko — CHANGES_REQUESTED** [review]

**Comment by aaronjudd** [comment]
Updated to METEOR@1.6.0.1 -> caught that after I committed it as well, but got distracted forgot to commit.

**Pull request closed without merging**